pkgs/development/python-modules/python-hwdata/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, hwdata
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "python-hwdata";
+  version = "2.4.1";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "xsuchy";
+    repo = "python-hwdata";
+    rev = "python-hwdata-${version}-1";
+    hash = "sha256-hmvxVF9LOkezXnJdbtbEJWhU4uvUJgxQHYeWUoiniF0=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
```

```suggestion
  nativeBuildInputs = [
    setuptools
  ];
```

Expand braces like that. Also for `pythonImportsCheck`.